server/locales/en.json

```diff
@@ -343,15 +351,47 @@
   "admin.groups.ruleDeny": "Deny",
   "admin.groups.ruleForceAllow": "Force Allow",
   "admin.groups.ruleLocales": "Locale(s)",
+  "admin.groups.ruleMatch": "Match",
   "admin.groups.ruleMatchEnd": "Path Ends With...",
   "admin.groups.ruleMatchExact": "Path Is Exactly...",
   "admin.groups.ruleMatchRegex": "Path Matches Regex...",
   "admin.groups.ruleMatchStart": "Path Starts With...",
   "admin.groups.ruleMatchTag": "Has Any Tag...",
   "admin.groups.ruleMatchTagAll": "Has All Tags...",
+  "admin.groups.rulePath": "Path",
   "admin.groups.ruleSites": "Site(s)",
   "admin.groups.ruleUntitled": "Untitled Rule",
   "admin.groups.rules": "Rules",
+  "admin.groups.rules.read.pages.hint": "Can view and search pages.",
+  "admin.groups.rules.read.pages.title": "Read Pages",
+  "admin.groups.rules.write.pages.hint": "Can create and edit pages.",
+  "admin.groups.rules.write.pages.title": "Write Pages",
+  "admin.groups.rules.review.pages.hint": "Can review and approve edits submitted by users.",
+  "admin.groups.rules.review.pages.title": "Review Pages",
+  "admin.groups.rules.manage.pages.hint": "Can move existing pages to other locations the user has write access to.",
+  "admin.groups.rules.manage.pages.title": "Manage Pages",
+  "admin.groups.rules.delete.pages.hint": "Can delete existing pages.",
+  "admin.groups.rules.delete.pages.title": "Delete Pages",
+  "admin.groups.rules.write.styles.hint": "Can insert CSS styles in pages.",
+  "admin.groups.rules.write.styles.title": "Use CSS",
+  "admin.groups.rules.write.scripts.hint": "Can insert JavaScript in pages.",
+  "admin.groups.rules.write.scripts.title": "Use JavaScript",
+  "admin.groups.rules.read.source.hint": "Can view pages source.",
+  "admin.groups.rules.read.source.title": "View Page Source",
+  "admin.groups.rules.read.history.hint": "Can view previous versions of pages.",
+  "admin.groups.rules.read.history.title": "View Page History",
+  "admin.groups.rules.read.assets.hint": "Can view / use assets (such as images and files) in pages.",
+  "admin.groups.rules.read.assets.title": "View Assets",
+  "admin.groups.rules.write.assets.hint": "Can upload new assets (such as images and files).",
+  "admin.groups.rules.write.assets.title": "Upload Assets",
+  "admin.groups.rules.manage.assets.hint": "Can edit and delete existing assets (such as images and files).",
+  "admin.groups.rules.manage.assets.title": "Manage Assets",
+  "admin.groups.rules.read.comments.hint": "Can view page comments.",
+  "admin.groups.rules.read.comments.title": "Read Comments",
+  "admin.groups.rules.write.comments.hint": "Can post new comments on pages.",
+  "admin.groups.rules.write.comments.title": "Write Comments",
+  "admin.groups.rules.manage.comments.hint": "Can edit and delete existing page comments.",
+  "admin.groups.rules.manage.comments.title": "Manage Comments",
```

Comment:
Same as above. use camelCase instead of dots, e.g. `admin.groups.ruleReadPagesHint`